joomla/joomla-cms
# Namespace JHtml

### Summary of Changes
~This is a backport of pr #17328 to Joomla 3. Additionally `JHtml` got namespaced and for demonstration purposes `JHtmlJquery`. If this one gets accepted, then I will namespace the rest of the services as well in another pr.~

Namespaces `JHtml`, why not the registry, se comments below.

### Testing Instructions
- Open the back end
- Check the source code of the page

### Expected result
All is working as expected.

### Actual result
All is working as expected.
